### PR TITLE
Fix duplication in FCM message.Added new integration test.

### PR DESCRIPTION
**Pull request by marian-craciunescu**
PN-358

### CODE REVIEW

**Review by bogh — APPROVED** [review]

**Review by bogh — APPROVED** [review]

**Comment by cosminrentea** [comment]
build failed

File: server/fcm/fcm_sender.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func fcmMessage(message *protocol.Message) *gcm.Message {
 			"messageID":      message.ID,
 			"correlation_id": message.CorrelationID(),
 		}).Error("Could not decode fcm.Message from guble message body")
-	} else if m.Notification != nil && m.Data != nil {
+	} else if m.Data != nil {
```
Comment (cosminrentea) [inline server/fcm/fcm_sender.go:51]: cool

**Review by cosminrentea — APPROVED** [review]

**Comment by cosminrentea** [comment]
triggered a rebuild

File: server/fcm_integration_test.go [hunk 2]
```diff
@@ -55,6 +55,10 @@ type expectedValues struct {
 
 // Test that restarting the service continues to fetch messages from store for a subscription from lastID
 func TestFCMRestart(t *testing.T) {
+
+	defer testutil.SkipIfDisabled(t)
+	defer testutil.SkipIfShort(t)
```
Comment (bogh) [inline server/fcm_integration_test.go]: Remove this. Just disabled.

**Review by bogh — APPROVED** [review]

**Review by bogh — APPROVED** [review]

**Review by bogh — APPROVED** [review]

**Pull request merged**